.golangci.yml

```diff
@@ -0,0 +1,54 @@
+run:
+  timeout: 10m
+  issues-exit-code: 1
+  tests: true
+  skip-dirs-use-default: true
+  modules-download-mode: readonly
+  allow-parallel-runners: false
+
+linters:
+  fast: false
+  enable:
+  - gofmt
+  - goimports
+  - golint
+  - gosec
+  - govet
+  - misspell
+  disable-all: false
+  presets:
+  - bugs
+  - unused
```

Comment:
There's a lot going on here and it's not clear to me why/how some of these settings were chosen. Does this originate from some kind of default or recommended configuration?

Reply:
it's a reasonable (to me) subset of checkers, culled from projects in the contour ecosystem and the golangci-lint defaults.